test/rage2e/rag_test.go

```diff
@@ -67,21 +71,83 @@ var _ = Describe("RAGEngine", func() {
 		}
 	})
 
+	It("should create RAG with localembedding and kaito VLLM workspace successfully", func() {
+		numOfReplica := 1
+		workspaceObj := createPhi3WorkspaceWithPresetPublicModeAndVLLM(numOfReplica)
+
+		time.Sleep(30 * time.Second)
+
+		validateWorkspaceResourceStatus(workspaceObj)
+
+		time.Sleep(30 * time.Second)
+
+		validateAssociatedService(workspaceObj.ObjectMeta)
+		validateInferenceConfigMap(workspaceObj)
```

Comment:
this validation is checked in vllm e2e test. you don't need to check it here?
just check the status of workplace.